src/ape/api/providers.py

```diff
@@ -159,6 +181,16 @@ def estimate_gas_cost(self, txn: TransactionAPI) -> int:
     def gas_price(self) -> int:
         ...
 
+    @property
+    @abstractmethod
+    def priority_fee(self) -> int:
+        ...
+
+    @property
+    @abstractmethod
+    def base_fee(self) -> int:
+        ...
+
```

Comment:
Should these raise `NotImplementedError`s by default? In case a network doesn't have EIP-1559

Reply:
I was imagining each of those networks to raise the error themselves, but yeah this makes it so they don't have to.

Done!